### ISSUE TITLE
Efficient mouseless functionality

### ISSUE THREAD

**Issue by WillSmithTE** [WillSmithTE/ameliorate#152]
**Is your feature request related to a problem? Please describe.**
I want to be able to exercise ameliorate's functionality without a mouse - both for accessibility, and for efficiency. Specifically, at least the following should be mouseless:
* Add node
* Edit node text
* ~Edit selection's score~ already done, type 0-9 or dash when selecting a part
* View score's claim
* Go back to problem mapping from claim view
* Select node up/down/left/right of current selection
* Select edge up/down/left/right of current selection

Note: most of these can be accomplished through [Vimium](https://github.com/philc/vimium)'s `[fF]`-to-click functionality, but that's not as smooth as directly having shortcuts for these (and requiring Vimium for accessibility is probably not ideal).

**Describe the solution you'd like**
VIM-mode sounds pretty awesome (e.g. "cc" to change node text, "cs" change score, "apj" add problem below, "<esc>" to enter VIM-mode). Perhaps more common hotkeys like ctrl-[char] would be good for non-vimmers too.

**Describe alternatives you've considered**

**Additional context**

**Technical ideas**
We already use [react-hotkeys-hook](https://github.com/JohannesKlauss/react-hotkeys-hook), which seems decent enough for this
